nixos/modules/system/boot/uvesafb.nix

```diff
@@ -0,0 +1,35 @@
+{ config, lib, pkgs, ... }:
```

Comment:
Yu can be a bit more liberal in spacing here. It is easier to read when using blank lines.

Reply:
More jump points with vim `}{` too :D

Reply:
ok, i think i got it

Reply:
I think I gott it right now. Coronavirus indentation xD